src/ansys/geometry/core/designer/component.py

```diff
@@ -495,6 +497,44 @@ def extrude_face(self, name: str, face: Face, distance: Union[Quantity, Distance
         self._master_component.part.bodies.append(tb)
         return Body(response.id, response.name, self, tb)
 
+    @protect_grpc
+    @check_input_types
+    @ensure_design_is_active
+    def create_sphere_body(self, name: str, center: Point3D, radius: Distance) -> Body:
+        """
+        Create a sphere body defined by the center point and the radius.
+
+        Parameters:
```

Comment:
@RobPasMue Is this formatting acceptable. It isn't using heading with underscores or omitting the initial articles per our usual standards.

Reply:
Hi @PipKat - no its not. @umutsoysalansys please align to proper docstring formatting